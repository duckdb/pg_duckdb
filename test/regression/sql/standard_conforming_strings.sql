CREATE TABLE foo(t text);
set standard_conforming_strings = off;
INSERT INTO foo VALUES('foo\'bar');
select * from foo where t = 'foo\'bar';

DROP TABLE foo;
