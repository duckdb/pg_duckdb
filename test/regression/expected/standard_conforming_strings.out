CREATE TABLE foo(t text);
set standard_conforming_strings = off;
INSERT INTO foo VALUES('foo\'bar');
WARNING:  nonstandard use of \' in a string literal
LINE 1: INSERT INTO foo VALUES('foo\'bar');
                               ^
HINT:  Use '' to write quotes in strings, or use the escape string syntax (E'...').
select * from foo where t = 'foo\'bar';
WARNING:  nonstandard use of \' in a string literal
LINE 1: select * from foo where t = 'foo\'bar';
                                    ^
HINT:  Use '' to write quotes in strings, or use the escape string syntax (E'...').
    t    
---------
 foo'bar
(1 row)

DROP TABLE foo;
